benchmark_problems/CUTEst/PALMER6C.dat

```diff
@@ -27,10 +27,10 @@ Data:          1 Response Variable  (y = KJmol-1)
 Model:         Miscellaneous Class
                8 Parameters (A0 to A14)
 
-               y = A0 + A2 * x**2 + A4 * x**4 + A6 ** x**6 + A8 * X**8 + A10 * x**10 + A12 * x**12 + A14 * x**14 +  e
+               y = A0 + A2 * x**2 + A4 * x**4 + A6 ** x**6 + A8 * x**8 + A10 * x**10 + A12 * x**12 + A14 * x**14 +  e
```

Comment:
Any particular reason for the new/removed blank lines here?

Reply:
I'm not sure what you mean about the new/removed blank lines but all I did was change
`A8 * X**8` to `A8 * x**8` since the fitters are case sensitive

Reply:
OK thanks that makes. Concerning the new/removed blank lines then these might be your editor. In the future if you do e.g. `git gui` or `git add -p` to check each change before a commit you see what I mean, or if you go to github and look in files changed you will see highlighted in green added lines and in red removed lines. In many cases this add/remove black lines, like this one where I now understand why they are likely there

Reply:
Do you mean the new/removed blank lines in the python files? This is due to the fact my editor is set to adhere  PEP8 formatting standards https://www.python.org/dev/peps/pep-0008/

Reply:
Sounds good - thanks